src/pages/journal/components/journal-navigation/__tests__/journal-navigation.spec.ts

```diff
@@ -65,10 +65,10 @@ describe('JournalNavigationComponent', () => {
     });
 
     describe('selected date is today', () => {
-      it('does not show previous day button', () => {
+      it('does show previous day button', () => {
         fixture.detectChanges();
         const previousDayContainer: DebugElement = componentEl.query(By.css('#previous-day-container'));
-        expect(previousDayContainer).toBeNull();
+        expect(previousDayContainer).toBeDefined();
```

Comment:
Careful of the distinction between `toBeNull()` and `toBeDefined()` when querying the component fixture. The absence of the queried selector in the DOM yields `null` which passes a check of `toBeDefined` - always use null when testing a query result.

Reply:
I'll be aware of this in the future. Thanks for the explanation.

Reply:
Refactored with `.not.toBeNull()`